### PR TITLE
Update the focus styles

### LINKED ISSUE #8543: Align focus and hover styles for indigo theme with the indigo design system
Implement all hover and focus styles from the design

## Additional context
Adobe XD [Design](https://xd.adobe.com/view/7dbb2644-85fb-44ce-b0b1-faaae862aa72-28e8/screen/bbf37315-599b-4efa-a068-436ceaa81645/specs/)

### PR DESCRIPTION
**Pull request by desig9stein**
Closes #8543

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by StefanIvanov** [comment]
The styling is all right I am changing the status to verified

File: projects/igniteui-angular/src/lib/core/styles/components/button-group/_button-group-theme.scss [hunk 1]
```diff
@@ -69,7 +69,8 @@
     $item-selected-text-color: null,
     $item-selected-background: null,
     $item-selected-border-color: null,
-    $item-selected-hover-background: null
+    $item-selected-hover-background: null,
+    $selected-shadow-color: null
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/button-group/_button-group-theme.scss:73]: You should document this property.

File: projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss [hunk 2]
```diff
@@ -109,6 +109,7 @@
     $flat-icon-color: null,
     $flat-hover-icon-color: null,
     $flat-focus-icon-color: null,
+    $flat-focus-outline: null,
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss:112]: Missing documentation.

File: projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss [hunk 3]
```diff
@@ -121,6 +122,7 @@
     $outlined-icon-color: null,
     $outlined-hover-icon-color: null,
     $outlined-focus-icon-color: null,
+    $outlined-focus-outline: null,
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss:125]: Missing documentation.

File: projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss [hunk 4]
```diff
@@ -131,6 +133,7 @@
     $raised-icon-color: null,
     $raised-hover-icon-color: null,
     $raised-focus-icon-color: null,
+    $raised-focus-outline: null,
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss:136]: Missing documentation.

File: projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss [hunk 5]
```diff
@@ -142,6 +145,7 @@
     $fab-hover-text-color: null,
     $fab-focus-background: null,
     $fab-focus-text-color: null,
+    $fab-focus-outline: null,
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss:148]: Missing documentation.

**Review by simeonoff — CHANGES_REQUESTED** [review]
Ok, but there are some newly added theme properties that remain undocumented.

**Review by simeonoff — APPROVED** [review]

**Pull request merged**